src/make/doSet.js

```diff
@@ -128,7 +129,10 @@ function doCard({card, cards, rawSetCards, code, set, baseSetSize}) {
     isDoubleFaced: isDoubleFaced,
     flippedCardURL: flippedCardURL,
     names: names,
-    supertypes: supertypes || []
+    supertypes: supertypes || [],
+    power,
+    toughness,
+    loyalty
```

Comment:
Is that something that should go here? https://github.com/dr4fters/dr4ft/pull/742